src/algebra/order.lean

```diff
@@ -96,10 +96,18 @@ lemma le_of_forall_lt [linear_order α] {a b : α}
   (H : ∀ c, c < a → c < b) : a ≤ b :=
 le_of_not_lt $ λ h, lt_irrefl _ (H _ h)
 
+lemma forall_lt_iff_le [linear_order α] {a b : α} :
+  (∀ ⦃c⦄, c < a → c < b) ↔ a ≤ b :=
+⟨le_of_forall_lt, λ h c hca, lt_of_lt_of_le hca h⟩
+
 lemma le_of_forall_lt' [linear_order α] {a b : α}
   (H : ∀ c, a < c → b < c) : b ≤ a :=
 le_of_not_lt $ λ h, lt_irrefl _ (H _ h)
 
+lemma forall_lt'_iff_le [linear_order α] {a b : α} :
```

Comment:
I'm not sure about the naming convention here, but I think the `'` goes on the whole name, not only on `lt`.

Reply:
Fixed.